caffe2/python/data_parallel_model.py

```diff
@@ -484,10 +484,24 @@ def FinalizeAfterCheckpoint(model, blobs=None):
     workspace.RunNet(model._checkpoint_net.Proto().name)
 
 
-def _Broadcast(devices, model, net, param):
+def _Broadcast(devices, model, net, param, use_nccl=False):
     # TODO(akyrola): replace with NCCLBroadcast when it's working
     # Copy params from gpu_0 to other
     master_gpu = devices[0]
+
+    if use_nccl:
+        if _IsGPUBlob(model, param):
+            master_device_opt = core.DeviceOption(caffe2_pb2.CUDA, master_gpu)
+            with core.DeviceScope(master_device_opt):
+                log.info('Use NCCL for Broadcast')
```

Comment:
I could have sworn there was an equivalent call for NCCL allreduce, which is why I put it in... Appears there isn't, so removed the output.